### PR TITLE
Parsing anchor-positioning fallback strategy

### PR DESCRIPTION
**Pull request by sanajaved7**

### CODE REVIEW

**Comment by sanajaved7** [comment]
[parseCSS() function](https://trello.com/c/XbX7dBVL/36-parsecss-function)

File: tests/parsing.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        { top: 'anchor(--button bottom)', right: 'anchor(--button right)' },
+        { bottom: 'anchor(--button top)', right: 'anchor(--button right)' },
+      ],
+    });
+  });
+
+  test('does not find @position-fallback at-rule', () => {
+    const result = parseCSS(cssWithoutAnchorPositioning);
+
+    expect(result.fallbackStrategy).toBeUndefined();
```
Comment (jgerigmeyer) [inline tests/parsing.test.ts:34]: I'm not sure this is testing anything meaningful, since AFAICT `fallbackStrategy` isn't a meaningful term, correct?

**Review by jgerigmeyer — APPROVED** [review]
Tests are passing! 🎉

File: tests/parsing.test.ts (hunk 1 above)
Comment (oluoluoxenfree) [reply tests/parsing.test.ts:34]: Yeah i'm fairly sure it doesn't do anything

**Pull request merged**